common/persistence/visibility/store/sql/query_converter_mysql.go

```diff
@@ -56,7 +56,9 @@ type (
 		JSONDoc2 sqlparser.Expr
 	}
 
-	mysqlQueryConverter struct{}
+	mysqlQueryConverter struct {
+		request *manager.ListWorkflowExecutionsRequestV2
```

Comment:
The less dependencies you have the better:
```suggestion
		nsID namespace.ID
```